src/interfaces/IFrontend.sol

```diff
@@ -3,7 +3,8 @@ pragma solidity ^0.8.23;
 
 interface IFrontend {
     /// @notice Platform address
-    function platform() external view returns (address);
```

Comment:
keep platform() to not have unneessacry change in interface, just ignore eslint rule in Frontend contract